angular-client/src/app/app.module.ts

```diff
@@ -190,19 +191,21 @@ import { BatteryInfoDisplay } from 'src/pages/charging-page/components/battery-i
     DynamicDialogModule,
     BrowserAnimationsModule,
     ButtonModule,
-    HttpClientModule,
     MatIconModule,
     SidebarModule,
     MatToolbarModule,
     MatButtonModule,
     MatInputModule,
     ReactiveFormsModule
   ],
-  providers: [DialogService, MessageService],
-  bootstrap: [AppContext]
+  providers: [
+    DialogService,
+    MessageService,
+    provideHttpClient(withInterceptorsFromDi())
+  ]
 })
 export class AppModule {
-  constructor(
+  constructor (
```

Comment:
This formatting issue will be addressed later by forcing formatting and linting in Git Hooks